draft-ietf-scitt-software-use-cases.md

```diff
@@ -236,23 +242,23 @@ Corresponding procedures are often called authenticated, measured, or secure boo
 The output of these high assurance boot procedures is often used as input to more complex verifications known as remote attestation procedures.
 
 If measurements before execution are not possible, static after-the-fact analysis is required, typically by examining artifacts.
-When best practices are followed, in such cases measurements (e.g., a hash or digests) are stored in a protected or shielded environment (e.g., TEEs or TPMs).
+When best practices are followed, measurements (e.g., a hash or digests) are stored in a protected or shielded environment (e.g., TEEs or TPMs).
 After finishing a boot sequence, these measurements about foundational firmware are retrieved after-the-fact from shielded locations and must be compared to reference values that are part of RIMs.
 A verifying system appraising the integrity of a boot sequence must identify, locate, retrieve, and authenticate corresponding RIMs.
 
-A consumer of published software wants:
+A consumer of published software wants to:
 
-- to easily identify sources for RIMs
-- to select appropriate RIMs and download them for the appraisal of measurements
-- to be able to assure the authenticity, applicability, and freshness of RIMs over time
+- easily identify sources for RIMs
+- select appropriate RIMs and download them for the appraisal of measurements
+- assure the authenticity, applicability, and freshness of RIMs over time
 
 There is no standardized way to:
 
 - identify, locate, retrieve and authenticate RIMs in a uniform fashion
-- to uniquely identify among multiple potential available RIMs (e.g., by age, source, signing authority, etc.)
-- to store RIMs in a fashion that enables their usage in appraisal procedures years after they were created in a secure and believable fashion
+- uniquely identify and filter multiple potential available RIMs (e.g., by age, source, signing authority, etc.)
+- store RIMs in a fashion that enables their usage in appraisal procedures years after they were created
```

Comment:
```suggestion
- store RIMs in a secure and believable fashion that enables their usage in appraisal procedures years after they were created
```

Reply:
not sure what "believable" means here, agree on secure.